aggrivate/aggrivate.js

```diff
@@ -748,4 +750,15 @@ function writeFeedFile(feedId, content) {
 function countFeedFiles() {
     var files = fs.readdirSync( '/path/to/feeds');
     return files.length;
-}
\ No newline at end of file
+}
+
+function getConfigFilePath() {
+    const configArgumentNameIndex = process.argv.findIndex((val) => (val === '--config' || val === '-c'));
```

Comment:
I think this script could benefit from using a package to perform the CLI argument parsing. I did this manually as I didn't want to introduce an external dependency.